xclim/testing/tests/test_indices.py

```diff
@@ -1365,62 +1365,51 @@ def test_simple(self, pr_series):
 
 
 class TestPrecipWettestDriestQuarter:
-    def test_exceptions(self, pr_series):
-        a = np.zeros(365 * 2)
-        a += 1 / 3600 / 24
-        a = pr_series(a, start="1971-01-01")
-        a[(a.time.dt.month == 9)] += 5 / 3600 / 24
-        a[(a.time.dt.month == 3)] += -1 / 3600 / 24
+    @staticmethod
+    def get_data(pr_series):
+        a = np.ones(731)
+        a = pr_series(a, start="1971-01-01", units="mm/d")
+        a[(a.time.dt.month == 9)] += 5
+        a[(a.time.dt.month == 3)] += -1
+        return a
 
+    def test_exceptions(self, pr_series):
+        a = self.get_data(pr_series)
         with pytest.raises(NotImplementedError):
-            xci.prcptot_wetdry_quarter(a, op="wettest", src_timestep="toto")
-        with pytest.raises(NotImplementedError):
-            xci.prcptot_wetdry_quarter(a, op="toto", src_timestep="D")
+            xci.prcptot_wetdry_quarter(a, op="toto")
 
     def test_simple(self, pr_series):
-        a = np.zeros(365 * 2)
-        a += 1 / 3600 / 24
-        a = pr_series(a, start="1971-01-01")
-        a[(a.time.dt.month == 9)] += 5 / 3600 / 24
-        a[(a.time.dt.month == 3)] += -1 / 3600 / 24
+        a = self.get_data(pr_series)
 
-        out = xci.prcptot_wetdry_quarter(a, op="wettest", src_timestep="D")
+        out = xci.prcptot_wetdry_quarter(a, op="wettest")
         np.testing.assert_array_almost_equal(out, [241, 241])
 
-        out = xci.prcptot_wetdry_quarter(a, op="driest", src_timestep="D")
+        out = xci.prcptot_wetdry_quarter(a, op="driest")
         np.testing.assert_array_almost_equal(out, [60, 60])
 
     def test_weekly_monthly(self, pr_series):
-        a = np.zeros(365 * 2)
-        a += 1 / 3600 / 24
-        a = pr_series(a, start="1971-01-01")
-        a[(a.time.dt.month == 9)] += 5 / 3600 / 24
-        a[(a.time.dt.month == 3)] += -1 / 3600 / 24
+        a = self.get_data(pr_series)
 
         p_weekly = xci.precip_accumulation(a, freq="7D")
         p_weekly.attrs["units"] = "mm week-1"
-        out = xci.prcptot_wetdry_quarter(p_weekly, op="wettest", src_timestep="W")
+        out = xci.prcptot_wetdry_quarter(p_weekly, op="wettest")
         np.testing.assert_array_almost_equal(out, [241, 241])
-        out = xci.prcptot_wetdry_quarter(p_weekly, op="driest", src_timestep="W")
+        out = xci.prcptot_wetdry_quarter(p_weekly, op="driest")
         np.testing.assert_array_almost_equal(out, [60, 60])
 
-        p_month = xci.precip_accumulation(a, freq="MS")
-        p_month.attrs["units"] = "mm month-1"
-        out = xci.prcptot_wetdry_quarter(p_month, op="wettest", src_timestep="M")
+        # Can't use precip_accumulation cause "month" is not a constant unit
```

Comment:
I tried the following and get only tiny differences  (1e-14): 
````
p_month = a.resample(time="MS").mean(keep_attrs=True)
p_month_amt = xci.precip_accumulation(a, freq="MS")
out = xci.prcptot_wetdry_quarter(p_month, op="wettest")
out1 =  xci.prcptot_wetdry_quarter(amount2rate(p_month), op="wettest")
np.testing.assert_array_equal(out, out1)  # error but really small
````

However pushing the limits with more data I see larger differences ( could have made a mistake I suppose) but I think these should result in the same value: 

````
np.random.seed(43)
a = np.random.uniform(low=0, high=35, size=365*50)  # ~50 yrs of random precip between 0, 35 mm)
a = pr_series(a, start="1971-01-01", units="mm/d")
for freq in ['MS', '3MS', 'YS', '5YS', '10YS', '20YS' ,'30YS']:
      print(freq)
       a1 = a.resample(time=freq).mean(keep_attrs=True)
       a2 = amount2rate(xci.precip_accumulation(a, freq=freq), out_units=a1.units)
       print(np.abs(a1-a2).max().values)

Reply:
Ok makes sense.